src/Social/Follows/Repositories/UsersFollowsRepository.php

```diff
@@ -28,4 +28,46 @@ public static function isFollowing(Users $user, EloquentModel $entity): bool
     {
         return (bool) self::getByUserAndEntity($user, $entity);
     }
+
+    /**
+     * getFollowers
+     */
+    public static function getFollowers(EloquentModel $entity): array
```

Comment:
@FredPeal you cant create the queries this way, it will use to much memory and not produce the results we want, you have two option move the logic to a builder like this https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/app/GraphQL/Inventory/Builders/Variants/Variants.php#L18 or this function return the builder itself not the result